test/levels.test.js

```diff
@@ -326,8 +348,11 @@ test('produces labels for custom levels', async ({ is }) => {
 test('setting changeLevelName does not affect labels when told to', async ({ is }) => {
   const instance = pino(
     {
-      useLevelLabels: true,
-      changeLevelName: 'priority'
```

Comment:
Can we keep supporting the `changeLevelName` in the default formatter with a deprecation warning?

Reply:
I'll see what I can do, given that Pino is the first customer of the new formatter option, supporting the old way for changing the levels and labels is not straightforward, I fear.